livekit-android-camerax/src/main/java/livekit/org/webrtc/CameraXHelper.kt

```diff
@@ -30,6 +30,18 @@ import io.livekit.android.room.track.video.CameraEventsDispatchHandler
 class CameraXHelper {
     companion object {
 
+        /**
+         * Creates a CameraProvider that uses CameraX for its sessions.
+         *
+         * For use with [CameraCapturerUtils.registerCameraProvider].
+         * Remember to unregister the provider when outside the lifecycle
+         * of [lifecycleOwner].
+         *
+         * @param lifecycleOwner The lifecycleOwner which controls the lifecycle transitions of the use cases.
+         */
+        @ExperimentalCamera2Interop
```

Comment:
I think adding `@JvmOverloads` to the original method would solve this. Would be a little cleaner that way.